python/tk_nuke/menu_generation.py

```diff
@@ -366,7 +366,7 @@ def _add_context_menu(self):
         action = ctx_menu.addAction("Jump to Shotgun")
         action.triggered.connect(self._jump_to_sg)
 
-        if self.context.filesystem_locations:
+        if ctx.filesystem_locations:
```

Comment:
This was a mistake in the original Jump To Filesystem implementation I submitted today. Should have tested in Hiero first!